server/images.py

```
@@ -316,6 +358,11 @@ def write_meta(self, image_id: str, changes: dict) -> bool:
         for key, value in changes.items():
             if key not in self.EDITABLE_FIELDS:
                 raise ValueError(f"field not editable: {key}")
+            if key in ("name", "folder") and str(value).strip():
+                # ‏#138: כאן עוברים גם `PUT /images/{id}` וגם שינוי שם
+                # תיקייה, ששניהם כותבים דרך write_meta.
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)